Shoko.Server/API/v3/Models/Shoko/Renamer.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Newtonsoft.Json;
+using Shoko.Models.Server;
+using Shoko.Server.API.v3.Models.Common;
+using Shoko.Server.Repositories;
+using Shoko.Server.Settings;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Linq;
+
+namespace Shoko.Server.API.v3.Models.Shoko
+{
+    public class Renamer : BaseModel
+    {
+        public Renamer(RenameFileHelper.RenamerInfo renamerInfo)
+        {
+            var scripts = RepoFactory.RenameScript.GetRenamerType(renamerInfo.Name);
+            ID = renamerInfo.ID;
```

Comment:
Int IDs for renamers faked in make no sense, and will cause possible client side cache bugs